src/content/reference/react/useSyncExternalStore.md

```diff
@@ -306,14 +307,14 @@ function subscribe(callback) {
 
 ---
 
-### Adding support for server rendering {/*adding-support-for-server-rendering*/}
+### Menambahkan dukungan untuk *render* di server {/*adding-support-for-server-rendering*/}
```

Comment:
```suggestion
### Menambahkan dukungan untuk render di server {/*adding-support-for-server-rendering*/}
```

For loan words, don't italicise it on headings. [(reference)](https://github.com/reactjs/id.react.dev/wiki/Panduan-Penulisan-Universal#panduan-penulisan-spesifik-bahasa-indonesia)